lib/matplotlib/colorbar.py

```diff
@@ -1274,6 +1298,111 @@ def _short_axis(self):
             return self.ax.xaxis
         return self.ax.yaxis
 
+    def _get_view(self):
+        """
+        Save information required to reproduce the current view.
+
+        The colorbar is controlled by the norm's vmin and vmax
+        """
+        return self.norm.vmin, self.norm.vmax
+
+    def _set_view(self, view):
+        """
+        Apply a previously saved view.
+
+        This sets the vmin and vmax of the norm associated with
+        the scalar mappable. We then update the normal to redraw
+        the content that was updated.
+        """
+        self.norm.vmin, self.norm.vmax = view
+        self.update_normal(self.mappable)
+
+    def _set_view_from_bbox(self, bbox, direction='in',
+                            mode=None, twinx=False, twiny=False):
+        """
+        Update view from a selection bbox.
+
+        The view of a colorbar is controlled by the vmin and vmax of the
+        norm. Updating the view from a bbox will change the vmin and vmax
+        to the extent of the selection when zooming in and scale the values
+        away from the selection when zooming out.
+
+        Parameters
+        ----------
+        bbox : 4-tuple or 3 tuple
+            * If bbox is a 4 tuple, it is the selected bounding box limits,
+              in *display* coordinates.
+            * If bbox is a 3 tuple, it is an (xp, yp, scl) triple, where
+              (xp, yp) is the center of zooming and scl the scale factor to
+              zoom by.
+        direction : str
+            The direction to apply the bounding box.
+                * `'in'` - The bounding box describes the view directly, i.e.,
+                           it zooms in.
+                * `'out'` - The bounding box describes the size to make the
+                            existing view, i.e., it zooms out.
+        mode : str or None
+            The selection mode, whether to apply the bounding box in only the
+            `'x'` direction, `'y'` direction or both (`None`).
+        twinx : bool
+            Whether this axis is twinned in the *x*-direction.
+        twiny : bool
+            Whether this axis is twinned in the *y*-direction.
+        """
+        new_xbound, new_ybound = self.ax._prepare_view_from_bbox(
+            bbox, direction=direction, mode=mode, twinx=twinx, twiny=twiny)
+        if self.orientation == 'horizontal' and not twinx and mode != "y":
+            self.norm.vmin, self.norm.vmax = new_xbound
+        if self.orientation == 'vertical' and not twiny and mode != "x":
+            self.norm.vmin, self.norm.vmax = new_ybound
+        self.update_normal(self.mappable)
+
+    def start_pan(self, x, y, button):
```

Comment:
This is just the same as `self.ax.start_pan` so can it just be `start_pan = self.ax.start_pan`?